app/src/main/java/com/example/trainingintive/what_to_do_feature/data/entity/ActivityModelFromApi.kt

```diff
@@ -0,0 +1,7 @@
+package com.example.trainingintive.what_to_do_feature.data.entity
+
+class ActivityModelFromApi(
```

Comment:
It will be best to use some name convention for all models in the data layer. So later on all models in the data layer should have `FromApi` suffix. So you can leave `FromApi` or change it to some other prefix/sufix.

Reply:
What suffix do you use in projects?

Reply:
`Remote` but you can use something else, depending what you prefer :)